config/storage_gcs_test.go

```diff
@@ -0,0 +1,101 @@
+package config
+
+import (
+	"reflect"
+	"testing"
+
+	storage "github.com/minamijoyo/tfmigrate-storage"
+	"github.com/minamijoyo/tfmigrate-storage/gcs"
+)
+
+func TestParseGCSStorageBlock(t *testing.T) {
+	cases := []struct {
+		desc   string
+		source string
+		want   storage.Config
+		ok     bool
+	}{
+		{
+			desc: "valid (required)",
+			source: `
+tfmigrate {
+  history {
+    storage "gcs" {
+      bucket = "tfmigrate-test"
+      name   = "tfmigrate/history.json"
+    }
+  }
+}
+`,
+			want: &gcs.Config{
+				Bucket: "tfmigrate-test",
+				Name:   "tfmigrate/history.json",
+			},
+			ok: true,
+		},
+		{
+			desc: "valid (with optional)",
```

Comment:
This case is the same as valid (required) and there is no optional attributes.

Reply:
@KengoTODA It looks the same as the above `valid (required)` case. I'm not sure what is different. Can we remove this case as duplicate?

Reply:
Yes it should be removed when I removed the `Endpoint` config. I will handle it soon.